app/models/account.rb

```diff
@@ -3,5 +3,7 @@ class Account < ApplicationRecord
 
   belongs_to :family
 
+  delegated_type :accountable, types: %w[Credit Depository Investment Loan OtherAsset OtherLiability Property Vehicle]
```

Comment:
Looks like we could reduce some duplication here?

```suggestion
  delegated_type :accountable, types: VALID_ACCOUNT_TYPES
```

Reply:
Yep, will push shortly.  Just in draft mode as I work through it.